awx/main/tasks.py

```diff
@@ -1212,7 +1221,8 @@ def run(self, pk, **kwargs):
                 )
                 ansible_runner.utils.dump_artifacts(params)
                 isolated_manager_instance = isolated_manager.IsolatedManager(
-                    cancelled_callback=lambda: self.update_model(self.instance.pk).cancel_flag
+                    cancelled_callback=lambda: self.update_model(self.instance.pk).cancel_flag,
+                    check_callback=self.check_callback,
```

Comment:
Think this should be `check_callback=self.check_handler`

Reply:
Right you are.  Fixed.